pkg/vm/engine/tae/options/types.go

```diff
@@ -55,8 +64,9 @@ type Options struct {
 	LogtailCfg    *LogtailCfg
 	Catalog       *catalog.Catalog
 
-	Clock clock.Clock
-	Fs    fileservice.FileService
-	Lc    logservice.Client
-	Shard metadata.DNShard
+	Clock     clock.Clock
+	Fs        fileservice.FileService
+	Lc        logservice.Client
```

Comment:
This should be a logservice create factory, and TAE will hold many log service instance to provide high performance.

Reply:
Ok, I'll use a create factory.